src/lib/common/sol-platform-impl-linux-micro.c

```diff
@@ -1001,3 +1013,55 @@ sol_platform_linux_micro_inform_service_state(const char *service, enum sol_plat
 #endif
     sol_platform_inform_service_monitors(service, state);
 }
+
+int
+sol_platform_impl_set_hostname(const char *name)
+{
+    return sol_platform_impl_linux_set_hostname(name);
+}
+
+const char *
+sol_platform_impl_get_hostname(void)
+{
+    return sol_platform_impl_linux_get_hostname();
+}
```

Comment:
IMO you should define `sol_platform_impl_set_hostname()` and `sol_platform_impl_get_hostname()` in the linux one, no need to create yet-another impl and call it from this one.